registry.json

```diff
@@ -1,3 +1,5 @@
 {
-  "dockstore" : "https://www.dockstore.org:8443"
+  "dockstore" : [
+      "https://www.dockstore.org:8443/api/v1/metadata"
+    ]
```

Comment:
I don't remember all the discussion from the call. If there are multiple URLs listed here, what's the significance? What order do they get processed in?